build_ngtf.py

```diff
@@ -38,12 +38,12 @@ def version_check(use_prebuilt_tensorflow):
     # Check bazel version
     bazel_ver = get_bazel_version()
     got_correct_bazel_version = False
-    if (int(bazel_ver[1]) >= 24 and int(bazel_ver[1]) <= 25):
+    if (int(bazel_ver[1]) > 24 and int(bazel_ver[1]) <= 25):
         if (int(bazel_ver[2]) >= 1 and int(bazel_ver[2]) <= 2):
             got_correct_bazel_version = True
 
     if not got_correct_bazel_version:
-        raise Exception("Need bazel 0.24.1 < version < 0.25.2 \n" + "Got: " +
+        raise Exception("Need bazel 0.24.x < version <= 0.25.2 \n" + "Got: " +
```

Comment:
Should it be 0.24.1 <= version <= 0.25.2

Reply:
But we want `0.24.1 < version <= 0.25.2`

Reply:
The check in 42 seems to be checking for =1 as well